### DISCUSSION TITLE
SERVICE_NOT_AVAILABLE

### DISCUSSION THREAD

**Discussion by NongKaittong** [invertase/react-native-firebase#5999] [Q&A] [answered]
How to fix Error: [messaging/unknown] java.io.IOException: java.util.concurrent.ExecutionException: java.io.IOException: SERVICE_NOT_AVAILABLE

react-native info
@react-native-firebase/messaging": "^14.2.1"

**Comment by andersonaddo** [comment, marked as answer]
Duplicate of https://github.com/invertase/react-native-firebase/issues/6000
